_includes/header.html

```diff
@@ -29,7 +29,12 @@ <h3><a href="/">FRC Team 1418 - George Mason High School</a></h3>
                     <li><a href="/robot/2014">2014</a></li>
                 </ul>
             </li>
-            <li><a href="/team">The Team</a></li>
+            <li><a href="/team">The Team</a>
+                <ul>
+                    <li>
+                        <a href="/alumni">Alumni</a>
+                    </li>
+                </ul>
```

Comment:
I think you need a closing "</li> tag right here. That way the <li> for the "The Team" link is closed.